Tools/_sunrise/test_pr_check_summary.py

```
@@ -0,0 +1,174 @@
+FailureDetails = pr_check_summary.FailureDetails
+build_comment = pr_check_summary.build_comment
+extract_failure_details = pr_check_summary.extract_failure_details
+latest_tracked_runs = pr_check_summary.latest_tracked_runs
+get_jobs_for_run = pr_check_summary.get_jobs_for_run
+rows_for_workflow = pr_check_summary.rows_for_workflow
+WorkflowConfig = pr_check_summary.WorkflowConfig
+JobConfig = pr_check_summary.JobConfig
+
+
+class PrCheckSummaryTests(unittest.TestCase):
+    def test_build_comment_renders_status_table_and_failure_details(self):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `test_build_comment_renders_status_table_and_failure_details`

Add return type annotation: `None`

(ANN202)